### PR TITLE
Add a way to access the wrapped component from `reduxForm` HoC

### PR DESCRIPTION
**Pull request by milkers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,8 +49,6 @@
 * [6.0.0-alpha.10](http://redux-form.com/6.0.0-alpha.10/)
 * [6.0.0-alpha.11](http://redux-form.com/6.0.0-alpha.11/)
 * [6.0.0-alpha.13](http://redux-form.com/6.0.0-alpha.13/)
-* [6.0.0-alpha.14](http://redux-form.com/6.0.0-alpha.14/)
-* [6.0.0-alpha.15](http://redux-form.com/6.0.0-alpha.15/)
```
Comment (ooflorent) [inline README.md]: Please revert these changes

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "redux-form",
-  "version": "5.2.5",
+  "name": "redux-form-t2",
```
Comment (ooflorent) [inline package.json]: Please revert these changes

**Comment by ooflorent** [comment]
Thanks for you contribution. The overall logic seems good to me. However:

* Could you revert the changes made to `README.md` and `package.json`?
* Could you run `npm run lint -- --fix` and fix the remaining errors?

**Comment by milkers** [comment]
Thank you for your advice. I tried to correct it as you like. Best.

**Comment by ooflorent** [comment]
Tests don't pass. Could you fix the linting?

**Comment by milkers** [comment]
Ok, I am on it. I will also commit the tests for `getWrappedInstance()`.

**Comment by lbdremy** [comment]
Nice addition I actually need this, how can I help to get this merge into the next release ?

**Comment by milkers** [comment]
Hey, thank you. I am having trouble to pass the codecov out of tests.

**Comment by milkers** [comment]
Hey @ooflorent, could you give some hints about how to pass tests while not losing from codecov?

**Comment by erikras** [comment]
Published in [`v5.3.4`](https://github.com/erikras/redux-form/releases/tag/v5.3.4).
